### DISCUSSION TITLE
Very slow or non existing refresh of BTC Price

### DISCUSSION THREAD

**Discussion by rolandwellinger** [BitMaker-hub/NerdMiner_v2#390] [General]
Hi All. I bought the device to use NerdMiner2 for showing stats. So Blockheight and Price and so. Everything refreshes good except of the btc price. this is days behind! And so unusable. Is there a way to improve the refresh of the btc price?

**Comment by paelzer** [comment]
Guess this is same issue than discussed here?

https://github.com/BitMaker-hub/NerdMiner_v2/issues/378

There is also good information how to fix it in the source code. 

Probably someone is already working on it to merge it into the master branch, not sure.

**Reply by rolandwellinger** [reply]
Oh sorry i didn't saw that. Perfect.  Thanks for the quick answer.
